cmake/onnxruntime_unittests.cmake

```diff
@@ -1158,7 +1158,6 @@ if (NOT onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
 
   # limit to only test on windows first, due to a runtime path issue on linux
```

Comment:
This comment needs to be changed, we're more than windows now right?

Reply:
yes, will send another pr to enable it.